static/app/views/performance/cache/tables/cacheHitMissCell.tsx

```diff
@@ -0,0 +1,15 @@
+import {t} from 'sentry/locale';
+import type {IndexedResponse} from 'sentry/views/starfish/types';
+
+function CacheHitMissCell(props: {hit: IndexedResponse['cache.hit']}) {
+  const {hit} = props;
+  if (hit === 'true') {
+    return <span>{t('HIT')}</span>;
+  }
+  if (hit === 'false') {
+    return <span>{t('MISS')}</span>;
+  }
+  return <span>--</span>;
```

Comment:
This is a good thing to dump in `settings.tsx`

Reply:
Sorry, i'm not sure what you mean, the entire Cell components?

Reply:
No no, just the `--` as the de facto missing cache description